src/valid/error.rs

```diff
@@ -125,6 +125,12 @@ impl From<serde_path_to_error::Error<serde_json::Error>> for ValidationError<Str
     }
 }
 
+impl From<anyhow::Error> for ValidationError<String> {
+    fn from(error: anyhow::Error) -> Self {
+        ValidationError::new(format!("Validation failed because of {}", error))
+    }
+}
```

Comment:
We should always from checked exception to unchecked ones. Here we are doing the opposite.